handlers/main.yml

```diff
@@ -20,3 +20,5 @@
   listen: restart-clickhouse
   register: clickhouse_restarted
   when: clickhouse_restart_on_changes
+  failed_when: clickhouse_restarted.failed and (not ansible_check_mode or not clickhouse_restarted.msg.startswith("Could not find the requested service"))
```

Comment:
Why check against `ansible_check_mode` is not enough?

Reply:
Just trying to limit ignored errors -- this way `--check` may help to prevent e.g. insufiicient privileges, or some other errors

Reply:
Yep, make sense (but if we will keep it, then we need a comment)